lib/internal/errors.js

```diff
@@ -1114,10 +1114,13 @@ E('ERR_INVALID_MODULE_SPECIFIER', (request, reason, base = undefined) => {
   return `Invalid module "${request}" ${reason}${base ?
     ` imported from ${base}` : ''}`;
 }, TypeError);
-E('ERR_INVALID_OPT_VALUE', (name, value) =>
-  `The value "${String(value)}" is invalid for option "${name}"`,
-  TypeError,
-  RangeError);
+E('ERR_INVALID_OPT_VALUE', (name, value, reason = '') => {
+  let inspected = typeof value === 'string' ?
+    value : lazyInternalUtilInspect().inspect(value);
+  if (inspected.length > 128) inspected = `${inspected.slice(0, 128)}...`;
+  if (reason) reason = '. Reason - ' + reason;
```

Comment:
```suggestion
  if (reason) reason = '. Reason: ' + reason;
```

Reply:
This was my initial implementation but after integrating it with validateOneOf it looked like `Reason: must be one of: 1, 2 3` which I didn't like so changed to a dash after reason.
Open to suggestions though for both of them.

Reply:
I would remove the colon after `:` so it was `Reason: must be one of 1, 2, 3`. But again, this is a total micro-nit pick and it can land without that change, or that change can happen in a later PR.

Reply:
Or even just remove `Reason`:  `Must be one of 1, 2, 3`.

Reply:
I think just removing `Reason` would be the best, we can always add it where needed. :+1: